joomla/joomla-cms
# [com_installer] Translate the extension name if possible

Pull Request for Issue #11567 
#### Summary of Changes

Allow all kind of extensions to have translatable names
#### Testing Instructions
- go to com_installer -> manage -> manage
- select the type `Library`
- see all items are translated (no language string)
- go to the database and edit the entry for the Joomla Lib
- set the name (collum params) to `LIB_JOOMLA_XML_DESCRIPTION`

`{"name":"LIB_JOOMLA_XML_DESCRIPTION","type":"library","creationDate":"2008","author":"Joomla! Project","copyright":"Copyright (C) 2005 - 2016 Open Source Matters. All rights reserved.","authorEmail":"admin@joomla.org","authorUrl":"https:\/\/www.joomla.org","version":"13.1","description":"LIB_JOOMLA_XML_DESCRIPTION","group":"","filename":"joomla"}`
- go to  com_installer -> manage -> manage again
- see it is not translated
- apply this PR
- see it is translated.
#### Documentation Changes Required

Nothing as this is the same as all other extensions.
